frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -45,7 +45,16 @@
       <div
         *ngIf="isEmpty$ | async"
         class="op-team-planner--no-data">
+        <img [src]="image.empty_state" class="op-team-planner--empty-state-image" />
         <span [textContent]="text.noData"></span>
+        <button
+          type="button"
+          class="op-team-planner--empty-state-button button -highlight"
+          data-qa-selector="op-team-planner--empty-state-button"
+          (click)="showAssigneeAddRow()"
+          [attr.aria-label]="text.add_assignee"
+          [textContent]="text.add_assignee"
```

Comment:
No need for a label if there's text, right?

Reply:
Yes probably not